### ISSUE TITLE
k zamysleni - otevreni formulare k editaci ne na cele plose

### ISSUE THREAD

**Issue by jzelenka** [zauzoo/fe#45]
k zamysleni - pri najeti mysi na formular v "zavrenem" rezimu, kdy formular zesedne, by se objevila nekde vpraco nahore ikona tuzky a pouze kliknuti na tuto ikonu by formular otevrelo. 

Duvody:
- ted nejde zkopirovat (ctrl+c) text ze zavreneho formulare
- na zavrenem formulari neni mozna dat linky (napriklad na party na detailu vehicle bych dal link na tuto party)

**Comment by milant007** [comment]
jo, ty by nebylo spatny, ale jak bys resil najeti mysi na mobilnim zarizeni?
